tests/bluetooth/tester/src/btp/btp_mcs.h

```diff
@@ -0,0 +1,31 @@
+/* btp_mcs.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/services/ots.h>
+
+/* MCS commands */
+#define BTP_MCS_READ_SUPPORTED_COMMANDS		0x01
+struct btp_mcs_read_supported_commands_rp {
+	uint8_t data[0];
+} __packed;
+
+#define BTP_MCS_CMD_SEND			0x02
+struct btp_mcs_send_cmd {
+	uint8_t opcode;
+	uint8_t use_param;
+	uint32_t param;
```

Comment:
The `param` in the MPL implementation is a `int32`. Should this be the same?

Reply:
Ah yes, missed that.